pkgs/tools/backup/mydumper/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub
+, cmake, glib, mariadb, libressl
+, pcre, pkgconfig, zlib
+}:
+
+stdenv.mkDerivation rec {
+  version = "0.9.3";
+  name = "mydumper-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "maxbube";
+    repo   = "mydumper";
+    rev    = "eef4d8291803e9ba57b4e58458cec3f4f7f0b203";
+    sha256 = "139v6707sxyslg7n1fii8b1ybdq50hbqhc8zf6p1cr3h2hhl6ns9";
+  };
+
+  buildInputs = [ cmake glib mariadb.client.dev libressl pcre pkgconfig zlib ];
```

Comment:
`cmake` and `pkgconfig` should be under `nativeBuildInputs` instead

Reply:
if changed to - nativebuildInputs = [ cmake pkgconfig ];
error build - make: *** No rule to make target 'install'.  Stop.